REPORT.md

```diff
@@ -341,16 +660,723 @@ vainfo: Supported profile and entrypoints
+|PICSIZE_B_Bossen_1|✔️|✔️|☠|☠|✔️|✔️|
+|PICSIZE_C_Bossen_1|✔️|✔️|✔️|☠|✔️|✔️|
+|PICSIZE_D_Bossen_1|✔️|✔️|✔️|☠|✔️|✔️|
+|PMERGE_A_TI_3|✔️|✔️|✔️|☠|✔️|✔️|
+|PMERGE_B_TI_3|✔️|✔️|✔️|☠|✔️|✔️|
+|PMERGE_C_TI_3|✔️|✔️|✔️|☠|✔️|✔️|
+|PMERGE_D_TI_3|✔️|✔️|✔️|☠|✔️|✔️|
+|PMERGE_E_TI_3|✔️|✔️|✔️|☠|✔️|✔️|
+|POC_A_Bossen_3|✔️|✔️|✔️|☠|✔️|✔️|
+|PPS_A_qualcomm_7|✔️|✔️|✔️|☠|✔️|✔️|
+|PS_B_VIDYO_3|✔️|✔️|✔️|☠|✔️|✔️|
+|RAP_A_docomo_6|✔️|✔️|✔️|☠|✔️|✔️|
+|RAP_B_Bossen_2|✔️|✔️|✔️|☠|✔️|✔️|
+|RPLM_A_qualcomm_4|✔️|✔️|✔️|☠|✔️|✔️|
+|RPLM_B_qualcomm_4|✔️|✔️|✔️|☠|✔️|✔️|
+|RPS_A_docomo_5|✔️|✔️|✔️|☠|✔️|✔️|
+|RPS_B_qualcomm_5|✔️|✔️|✔️|☠|✔️|✔️|
+|RPS_C_ericsson_5|✔️|✔️|✔️|☠|✔️|✔️|
+|RPS_D_ericsson_6|✔️|✔️|❌|☠|✔️|✔️|
+|RPS_E_qualcomm_5|✔️|✔️|❌|☠|✔️|✔️|
+|RPS_F_docomo_2|✔️|✔️|✔️|☠|✔️|✔️|
+|RQT_A_HHI_4|✔️|✔️|✔️|☠|✔️|✔️|
+|RQT_B_HHI_4|✔️|✔️|✔️|☠|✔️|✔️|
+|RQT_C_HHI_4|✔️|✔️|✔️|☠|✔️|✔️|
+|RQT_D_HHI_4|✔️|✔️|✔️|☠|✔️|✔️|
+|RQT_E_HHI_4|✔️|✔️|✔️|☠|✔️|✔️|
+|RQT_F_HHI_4|✔️|✔️|✔️|☠|✔️|✔️|
+|RQT_G_HHI_4|✔️|✔️|✔️|☠|✔️|✔️|
+|SAO_A_MediaTek_4|✔️|✔️|✔️|☠|✔️|✔️|
+|SAO_B_MediaTek_5|✔️|✔️|✔️|☠|✔️|✔️|
+|SAO_C_Samsung_5|✔️|✔️|✔️|☠|✔️|✔️|
+|SAO_D_Samsung_5|✔️|✔️|✔️|☠|✔️|✔️|
+|SAO_E_Canon_4|✔️|✔️|✔️|☠|✔️|✔️|
+|SAO_F_Canon_3|✔️|✔️|✔️|☠|✔️|✔️|
+|SAO_G_Canon_3|✔️|✔️|✔️|☠|✔️|✔️|
+|SAO_H_Parabola_1|✔️|✔️|✔️|☠|✔️|✔️|
+|SAODBLK_A_MainConcept_4|❌|✔️|✔️|☠|✔️|✔️|
+|SAODBLK_B_MainConcept_4|❌|✔️|✔️|☠|✔️|✔️|
+|SDH_A_Orange_4|✔️|✔️|✔️|☠|✔️|✔️|
+|SLICES_A_Rovi_3|✔️|✔️|✔️|☠|✔️|✔️|
+|SLIST_A_Sony_5|✔️|✔️|✔️|☠|✔️|✔️|
+|SLIST_B_Sony_9|✔️|✔️|✔️|☠|✔️|✔️|
+|SLIST_C_Sony_4|✔️|✔️|✔️|☠|✔️|✔️|
+|SLIST_D_Sony_9|✔️|✔️|✔️|☠|✔️|✔️|
+|SLPPLP_A_VIDYO_2|✔️|✔️|✔️|☠|✔️|✔️|
+|STRUCT_A_Samsung_7|✔️|✔️|✔️|☠|✔️|✔️|
+|STRUCT_B_Samsung_7|✔️|✔️|✔️|☠|✔️|✔️|
+|TILES_A_Cisco_2|✔️|✔️|✔️|☠|✔️|✔️|
+|TILES_B_Cisco_1|✔️|✔️|✔️|☠|✔️|✔️|
+|TMVP_A_MS_3|✔️|✔️|✔️|☠|✔️|✔️|
+|TSCL_A_VIDYO_5|✔️|✔️|✔️|☠|✔️|✔️|
+|TSCL_B_VIDYO_4|✔️|✔️|✔️|☠|✔️|✔️|
+|TSKIP_A_MS_3|✔️|✔️|✔️|☠|✔️|✔️|
+|TSUNEQBD_A_MAIN10_Technicolor_2|☠|☠|☠|☠|❌|✔️|
+|TUSIZE_A_Samsung_1|✔️|✔️|✔️|☠|✔️|✔️|
+|VPSID_A_VIDYO_2|✔️|✔️|✔️|☠|✔️|✔️|
+|VPSSPSPPS_A_MainConcept_1|❌|❌|❌|☠|❌|✔️|
+|WP_A_MAIN10_Toshiba_3|✔️|✔️|✔️|☠|✔️|✔️|
+|WP_A_Toshiba_3|✔️|✔️|✔️|☠|✔️|✔️|
+|WP_B_Toshiba_3|✔️|✔️|✔️|☠|✔️|✔️|
+|WP_MAIN10_B_Toshiba_3|✔️|✔️|✔️|☠|✔️|✔️|
+|WPP_A_ericsson_MAIN10_2|✔️|✔️|✔️|☠|✔️|✔️|
+|WPP_A_ericsson_MAIN_2|✔️|✔️|✔️|☠|✔️|✔️|
+|WPP_B_ericsson_MAIN10_2|✔️|✔️|✔️|☠|✔️|✔️|
+|WPP_B_ericsson_MAIN_2|✔️|✔️|✔️|☠|✔️|✔️|
+|WPP_C_ericsson_MAIN10_2|✔️|✔️|✔️|☠|✔️|✔️|
+|WPP_C_ericsson_MAIN_2|✔️|✔️|✔️|☠|✔️|✔️|
+|WPP_D_ericsson_MAIN10_2|✔️|✔️|❌|☠|✔️|✔️|
+|WPP_D_ericsson_MAIN_2|✔️|✔️|❌|☠|✔️|✔️|
+|WPP_E_ericsson_MAIN10_2|✔️|✔️|✔️|☠|✔️|✔️|
+|WPP_E_ericsson_MAIN_2|✔️|✔️|✔️|☠|✔️|✔️|
+|WPP_F_ericsson_MAIN10_2|✔️|✔️|✔️|☠|✔️|✔️|
+|WPP_F_ericsson_MAIN_2|✔️|✔️|✔️|☠|✔️|✔️|
+|-|-|-|-|-|-|-|
+|TOTAL|143/147|145/147|137/147|0/147|143/147|147/147|
+
+
+### H.264/AVC Nvidia
+
+`./fluster.py run -d JCT-VT-H.264 GStreamer-H.264-VAAPI-Gst1.0 FFmpeg-H.264-VAAPI GStreamer-H.264-NVDEC-Gst1.0 FFmpeg-H.264-VDPAU FFmpeg-H.264 -ts JVT-AVC_V1 -s`
```

Comment:
Same as for H265 decoder previously, remove VAAPI based ones as we don't know which GPU they are using internally.